src/deb/bintray.json

```diff
@@ -0,0 +1,33 @@
+{
+    "package": {
+        "name": "chronos",
+        "repo": "paasta",
+        "subject": "yelp",
+        "desc": "Chronos is a replacement for cron. It is a distributed and fault-tolerant scheduler that runs on top of Apache Mesos that can be used for job orchestration.",
+        "website_url": "https://mesos.github.io/chronos/",
+        "issue_tracker_url": "https://github.com/Yelp/mesos/issues",
+        "vcs_url": "https://github.com/Yelp/chronos.git",
+        "github_use_tag_release_notes": true,
+        "github_release_notes_file": "RELEASE.md",
+        "licenses": ["Apache-2.0"],
+        "labels": [],
+        "public_download_numbers": false,
+        "public_stats": false,
+        "attributes": []
+    },
+
+    "version": {
+        "name": "${project.version}",
+        "desc": "Version ${project.version}",
+        "released": "${build.timestamp}",
+        "vcs_tag": "v${project.version}",
+        "attributes": [],
+        "gpgSign": false
+    },
+
+    "files":
+        [
+        {"includePattern": "target/(.*.deb)", "uploadPattern": "$1", "matrixParams": { "deb_distribution": "lucid,trusty", "deb_component": "main", "deb_architecture": "amd64"} }
```

Comment:
actually, I guess that gets harder because you'll need to do special things for systemd. never mind.